Applications/CMC/test/testCMCArm26_Millard.cpp

```diff
@@ -45,7 +45,7 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    std::vector<double> rms_tols(2*2+2*6, 0.02); // activations within 2%, angles within .6 degrees
```

Comment:
I know you didn't put it there, but the trailing comment is obsolete and should be deleted.